build.zig

```diff
@@ -376,24 +387,21 @@ pub fn build(b: *Build) !void {
         for (sys_libs.items) |slib| {
             std_lib.linkSystemLibrary(slib);
         }
-        if (!build_options.use_mimalloc) {
+        if (build_options.needLibC()) {
             std_lib.linkLibC();
         }
         std_lib.setMainPkgPath("src");
         std_lib.linkLibrary(lib);
         std_lib.addOptions("build_options", build_options.step(b));
 
         // Adds `$BUZZ_PATH/lib` and `/usr/local/lib/buzz` as search path for other shared lib referenced by this one (libbuzz.dylib most of the time)
-        var buzz_path = std.ArrayList(u8).init(std.heap.page_allocator);
-        defer buzz_path.deinit();
-        buzz_path.writer().print(
+        std_lib.addRPath(b.fmt(
             "{s}{s}lib",
             .{
-                std.os.getenv("BUZZ_PATH") orelse std.fs.cwd().realpathAlloc(std.heap.page_allocator, ".") catch unreachable,
+                std.os.getenv("BUZZ_PATH") orelse std.fs.cwd().realpathAlloc(b.allocator, ".") catch unreachable,
```

Comment:
Does this work the first time we build? Since `BUZZ_PATH` is only set L446 ?

Reply:
I think so.